logging/handlers.ts

```diff
@@ -0,0 +1,65 @@
+import { open, File, Writer } from "deno";
+import { getLevelByName } from "./levels.ts";
+import { LogRecord } from "./logger.ts";
+
+export class BaseHandler {
+  level: number;
+  levelName: string;
+
+  constructor(levelName: string) {
+    this.level = getLevelByName(levelName);
+    this.levelName = levelName;
+  }
+
+  handle(logRecord: LogRecord) {
+    if (this.level > logRecord.level) return;
+
+    // TODO: implement formatter
```

Comment:
Maybe use deno.inspect ?

Reply:
I had I mind something like: 
```
{levelName} [{datetime}] {msg}
``` 
which gives: 
```
DEBUG [2018-12-30T20:11:53] Some message
``` 

With some sane default.

Reply:
I see. Another thought: can you use JS template literals? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
I am not sure I understand. If we use template strings I won't be possible to format arguments. Can you give an example what you had in mind?

Reply:
Hmm. I don't have something specific in mind now.  JS has some nice functionality now for doing formatting and it would be good to take advantage of that if possible. I'll get back to you on that...

Reply:
Sure, I'm curious